src/controllers/hotkeys/ActionNames.hpp

```diff
@@ -195,6 +195,18 @@ inline const std::map<HotkeyCategory, ActionDefinitionMap> actionNames{
               },
               .argumentsPrompt = "Target popup:",
           }},
+         {"setHighlightSounds",
+          ActionDefinition{
+              .displayName = "Set highlight sounds",
+              .argumentDescription = "[on or off. default: toggle]",
+              .minCountArguments = 0,
+              .maxCountArguments = 1,
+              .possibleArguments = HOTKEY_ARG_ON_OFF_TOGGLE,
+              .argumentsPrompt = "New value:",
+              .argumentsPromptHover =
+                  "Should highlight sounds be enabled, disabled or toggled",
+          }},
+         {"subscribe", ActionDefinition{"Open subscription page"}},
```

Comment:
bikeshed: "subscribe" feels wrong (I know that's from the split header). Just from the name, I wouldn't know what it does (of course, if you know what it does, thinking about what you think it would do if you didn't know is hard). When I read the description, I wasn't sure what "subscribe" did. Maybe `openSubscriptionPage`?